providers.tf.example

```diff
@@ -14,16 +14,22 @@ provider "aws" {
 provider "kubernetes" {
   host                   = data.aws_eks_cluster.cluster.endpoint
   cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority[0].data)
-  token                  = data.aws_eks_cluster_auth.cluster.token
+
+  # https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs#exec-plugins
+  exec {
+    api_version = "client.authentication.k8s.io/v1beta1"
+    args        = ["eks", "get-token", "--profile", "change_me", "--cluster-name", data.aws_eks_cluster.cluster.name]
+    command     = "aws"
+  }
 }
 
 provider "helm" {
   kubernetes {
     host                   = data.aws_eks_cluster.cluster.endpoint
     cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority[0].data)
     token                  = data.aws_eks_cluster_auth.cluster.token
-    
-    # Uncomment if you run into Helm timeout issues on Linux 
+
+    # Uncomment if you run into Helm timeout issues on Linux
     #exec {
     #  api_version = "client.authentication.k8s.io/v1beta1"
     #  args        = ["eks", "--profile=change_me", "get-token", "--cluster-name", data.aws_eks_cluster.cluster.name]
```

Comment:
Same as previous comment.

Reply:
I just installed the auto-formatting feature in VSCode as _recommended_ in the contributing guidelines. Seems the auto-formatter just removed some useless spaces. Is this a problem?

Reply:
Not that, I'm referring on change_me name of profile usage. I think that we should setup profile name in one place in file and then reuse value.